app/assets/stylesheets/styles/components/pages/_home.css.scss

```diff
@@ -68,58 +56,27 @@ video { display: block; }
   }
 
   h2 {
+    color: #ffffff;
```

Comment:
Probably, in case we decide white isn't the color for us, we don't want to go hunt all of these white text things down.